webpack/assets/javascripts/react_app/components/AutoComplete/components/AutoCompleteClearButton.js

```diff
@@ -5,31 +5,26 @@ import { Icon, OverlayTrigger, Tooltip } from 'patternfly-react';
 import { noop } from '../../../common/helpers';
 import { translate as __ } from '../../../common/I18n';
 
-const AutoCompleteClearButton = ({ onClear, tooltipID }) => {
-  const tooltip = <Tooltip id={tooltipID}>{__('Clear')}</Tooltip>;
-  return (
-    <OverlayTrigger
-      overlay={tooltip}
-      placement="top"
-      trigger={['hover', 'focus']}
-    >
-      <Icon
-        name="close"
-        className="autocomplete-clear-button"
-        onClick={onClear}
-      />
-    </OverlayTrigger>
-  );
-};
+const AutoCompleteClearButton = ({ onClear }) => (
+  <OverlayTrigger
+    overlay={<Tooltip id={UUID()}>{__('Clear')}</Tooltip>}
```

Comment:
It is better to generate a new `uuid` in every for each render then use it as a `default-prop`.
when uses as `default-prop` it produce the same id for all the instances.

Reply:
an alternative is to set it in a class constructor